Makefile

```diff
@@ -120,7 +120,11 @@ AWS_MAX_ATTEMPTS ?= 300
 PACKAGE_VERSIONS ?= ""
 SALT_VERSION ?= $(shell ./scripts/get-salt-version.sh $(BASE_NAME) $(STACK_VERSION))
 SALT_PATH ?= /opt/salt_$(SALT_VERSION)
-PYZMQ_VERSION ?= 19.0
+ifeq ($(SALT_VERSION),3006.4)
```

Comment:
Is there a reason on the strict match? I think next salt versions will need at least 25.0.2 instead of 19.0...

Reply:
I do not know yet what version of pyzmq will be needed for newer salt versions so I left it like this

Reply:
Yes, I agree you don't know what will be needed, but having 25.0.2 is a better guess then 19.0 so in the happy scenario you won't need to touch it again if you use 25.0.2 for salt 3006.4 and higher versions.

Reply:
Fixed it
Test runs (aborted after PYZMQ_VERSION calculation):
http://build.eng.hortonworks.com/job/cloudbreak-multi-packer-image-builder-salt-v3/3280/
http://build.eng.hortonworks.com/job/cloudbreak-multi-packer-image-builder-salt-v3/3279/